api/envoy/api/v2/auth/cert.proto

```diff
@@ -443,6 +443,11 @@ message DownstreamTlsContext {
     lt {seconds: 4294967296}
     gte {}
   }];
+
+  // If set to true, TLS session tickets are disabled. Setting this field to true and configuring either one of :ref:`session_ticket_keys
+  // <envoy_api_field_auth.DownstreamTlsContext.session_ticket_keys>` or :ref:`session_ticket_keys_sds_secret_config
+  // <envoy_api_field_auth.DownstreamTlsContext.session_ticket_keys_sds_secret_config>` will result in an error.
+  bool disable_session_tickets = 7;
```

Comment:
Sorry for the confusion on this; this field should still be part of `oneof session_ticket_keys_type`, it should just be numbered 7. Note that the field numbers do NOT have to be in order in a message in a .proto.